Rakefile

```diff
@@ -1,4 +1,6 @@
+require 'bundler/gem_tasks'
```

Comment:
We don't really need it, but I thought it was a nice to have, since doing `rake release` will take care of everything (tagging and pushing).
I can remove the dependency, if you prefer.